.travis.yml

```diff
@@ -2,9 +2,22 @@ services:
   - docker
 language: bash
 
+env:
+  global:
+    - REGISTRY_USER=praekeltorgdeploy
+    - secure: "b6edRlN/D6k7Vqm3LGti9khu2zNV+BvQKQigBmzpTsjq/C2W1m82WN/+95Ck91jogFGPWndz+L6NKi1t1SefVHhJAvz8rC7drF7b1KcOx3fA8HVvkCsqOX3YEl27dCTA6TDna4decW3A+5MQn6l11ksIVPArRux9jLKPRbUUAsauUTYPZYQ7LaUXA/rW7IQWDegoGTUW0CXJwtT8xhrnmw814tG5j5DRPh4RXfw9uitCuBu2k6/g3fus2eX8FGZUzznLCqul4ruXECuIOFk+WZ4f3q3rauVy5CQsjleTJB3Iz+RiH9zRXZYmJVGr8gSoVCgdF3Bg1FBMeK1SRd7BjPPzV4XI8PgduDNwgXM4iBx7QJt8GM0ygYJWBN/mKZ8q4bGAD3bfPMemgPJmvJXPxhX/obEJb+3epbTq3I7iaF4+AgszZszBz62hmNtdpRCmXhZftcKTLnIcoEUfNxkYTlS78xcCv6rbsDHLf0O07WlRBE3pSJNhnwwykBVsAhEx3MFvxrVu6X3ut+/FEGky6vtN8GD1O/ibIfzw/C7d1uyGpjp/vvKDTOHb/oB8qj3ptql1KRHZIBWh5ZeUyblLH8k4cPKbk+PjJFqMhF72MgTdMzcKls1CrV/2VMnp//c8wAgDQN3e2aehkswqvu3rffWT9yxJg6EdHTITldJ9eJg="
+
 script:
-  - docker-compose build
+  - docker build --tag praekeltfoundation/junebug .
```

Comment:
Originally I did this because docker-compose was building the image and giving it its own generated tag. After 47db6a1 this is basically the same thing as doing a `docker-compose build` as docker-compose will give it the tag we want.

I'm leaving it like this, though, so that all the tagging can be seen in the Travis file.